server/routes/v1/post.ts

```diff
@@ -7,6 +7,7 @@ const postRouter = express.Router();
 const postController = new PostController(db.Post, db.UserPostLikes);
 
 postRouter.get('', async (req: Request, res: Response) => {
```

Comment:
create a `/search` endpoint.

Reply:
That was actually what I did first, but it ended up conflicting with the /posts/:postid/ endpoints when you actually call something like uboard.com/posts/search

Reply:
Nvm figured it out, just had to move the route to the top of the file (ahove the :postid routes)